client/webserver/site/src/js/locales.ts

```diff
@@ -93,6 +94,7 @@ export const enUS: Locale = {
   [ID_ESTIMATE_UNAVAILABLE]: 'estimate unavailable',
   [ID_NO_ZERO_RATE]: 'zero rate not allowed',
   [ID_NO_ZERO_QUANTITY]: 'zero quantity not allowed',
+  [ID_NO_QUANTITY_EXCEEDS_MAX]: 'not enough funds',
```

Comment:
I'm not sure what needs to be done about translations, or will it pick English by default if the local one isn't present ?

Reply:
Yes, it will.